packages/Webkul/API/Http/Controllers/Shop/AddressController.php

```diff
@@ -104,26 +106,28 @@ public function store()
      *
      * @return \Illuminate\Http\Response
      */
-    public function update()
+    public function update($id)
     {
         $customer = auth($this->guard)->user();
```

Comment:
Just check this variable in the update method only, I think this variable is not used anywhere so not needed.

Reply:
It's used to take the id from the route and pass it to the repository instead of looking for the id in the request.

Reply:
I am referring to `$customer`.